examples/main.cpp

```
@@ -0,0 +1,88 @@
+int main(int argc, const char **argv)
+{
+  auto options_opt = mr::CliOptions::parse(argc, argv);
+  if (options_opt == std::nullopt) {
+    return 1;
+  }
+  auto options = *options_opt;
+  options.print();
+
+  mr::Application app(options.mode == mr::CliOptions::Mode::Default);
+
+  mr::Extent render_context_extent = {};
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)